From 878a78f86d7c6127fa5dfa55cddc3f74c62a1b6f Mon Sep 17 00:00:00 2001
From: Emmanuele Bassi <ebassi@gnome.org>
Date: Mon, 17 Jul 2017 20:08:58 +0100
Subject: [PATCH] Drop parallel builds

Sadly, autotools are bad, and generated files are a problem in 2017.
---
 Makefile.am | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index 9fd2bc6..1242983 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -93,7 +93,10 @@ ENUMS_FILES =					\
 	$(NULL)
 
 $(GCAB_ENUMS): %: %.etemplate $(ENUMS_FILES)
-	$(AM_V_GEN)$(GLIB_MKENUMS) --template $^ | sed -e 's/g_cab/gcab/g' | sed -e 's/G_CAB/GCAB/g' | sed -e 's/_CAB//g' > $@
+	$(AM_V_GEN)$(GLIB_MKENUMS) --template $^ | \
+		sed -e 's/g_cab/gcab/g' \
+		    -e 's/G_CAB/GCAB/g' \
+		    -e 's/_CAB//g' > $@
 
 EXTRA_DIST =					\
 	gcab.doap				\
@@ -222,4 +225,6 @@ $(top_srcdir)/.version:
 dist-hook:
 	echo $(VERSION) > $(distdir)/.tarball-version
 
+.NOTPARALLEL:
+
 -include $(top_srcdir)/git.mk
-- 
2.13.0

